person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
```

Comment:
- [ ] Please as the last reviewer said Kindly ensure that the Person class inherits from the Nameable class and it also has a method correct_name as specified by the project requirement 👍🏽